.github/SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Policy
+
+If you discover any security related issues, please email craig.paul@coconutcalendar.com instead of using the issue tracker.
```

Comment:
Can you change the email to `@coconutsoftware.com` (it still forwards, but this is more accurate)